### PR TITLE
Ab#46599, Ab#46600: Updated public api documentation for add/edit subject with height info

### PR DESCRIPTION
**Pull request by karuna-bhat**
Ab#46599, Ab#46600, Ab#32625, Ab#32668

## Changes Made
- Updated public api documentation for add/edit subject with height info

### CODE REVIEW

**Review by mark-fogle — APPROVED** [review]

File: sections/subjects.md [hunk 1]
```diff
@@ -200,12 +208,14 @@ dob|ISO8601 Date||day before present day|Yes|||must be day before present day
 gender|String|||Yes|<ul><li>Male</li><li>Female</li></ul>||Study/site shall be configured to utilize this field
 weight|Number|1|1000|Yes|||Study/site shall be configured to utilize this field
 weightUnit|String|||Yes|lbs or kg||Study/site shall be configured to utilize this field
+height|Number|1|300|Yes (if allowed by site)|||Study/site shall be configured to utilize this field
+heightUnit|String|||Yes (if allowed by site)|cm or inches||
```
Comment (mark-fogle) [inline sections/subjects.md:212]: This says cm or inches. Is it "inches" or "in"?

Comment (mark-fogle) [reply sections/subjects.md:212]: I am also looking at Amulya's PR which says "in". So, maybe there is some inconsistency there?

https://github.com/actigraph/CentrePoint3APIDocumentation/pull/34

Comment (mark-fogle) [reply sections/subjects.md:212]: The DTO there is specified like this:
```csharp
 public enum HeightUnitDto
    {
        [EnumMember(Value = "cm")]
        cm = 1,

        [EnumMember(Value = "in")]
        inches = 2
    }
```

Maybe the other is different

Comment (karuna-bhat) [reply sections/subjects.md]: @timgriffeth Just corrected the Public one and have included it in the [PR](https://github.com/actigraph/centrepoint-3-api/pull/277).

File: sections/subjects.md [hunk 2]
```diff
@@ -200,12 +208,14 @@ dob|ISO8601 Date||day before present day|Yes|||must be day before present day
...
+heightUnit|String|||Yes (if allowed by site)|cm or in||
 wearPosition|String|||Yes|<ul><li>Non-Dominant Wrist</li><li>Dominant Wrist</li><li>Left Non-Dominant Wrist</li><li>Left Dominant Wrist</li><li>Right Non-Dominant Wrist</li><li>Right Dominant Wrist</li><li>Waist</li><li>Left Wrist</li><li>Right Wrist</li><li>Ankle</li></ul>||Study/site shall be configured to utilize this field
 changeReason|String|||Yes|||Study/site shall be configured to utilize this field. Captured in operator audit record in accordance  with FDA 21 CFR Part 11.
 
 **Additional Notes:**
 
-- Depending on the study/site configuration of subject being edited, the **Gender**, **DOB**, and/or **Weight** fields may or may not be allowed. If the fields are allowed, then they will be required. If not allowed, then these fields must be excluded from the JSON request.
+- Depending on the study/site configuration of subject being edited, the **Gender**, **DOB**, and/or **Weight**, and/or **Height** fields may or may not be allowed. If the fields are allowed, then they will be required. If not allowed, then these fields must be excluded from the JSON request.
```
Comment (timgriffeth) [inline sections/subjects.md:218]: Lets remove the first and/or between dob and weight.

**Review by timgriffeth — APPROVED** [review]

File: sections/subjects.md [hunk 3]
```diff
@@ -133,11 +135,13 @@ dob|ISO8601 Date||day before present day|Yes (if allowed by site)|||must be day
...
 weightUnit|String|||Yes (if allowed by site)|lbs, kg||
+height|Number|1|300|Yes (if allowed by site)|||Study/site shall be configured to utilize this field
+heightUnit|String|||Yes (if allowed by site)|cm, in||
 wearPosition|String|||Yes (if changes allowed by study)|<ul><li>Non-Dominant Wrist</li><li>Dominant Wrist</li><li>Left Non-Dominant Wrist</li><li>Left Dominant Wrist</li><li>Right Non-Dominant Wrist</li><li>Right Dominant Wrist</li><li>Waist</li><li>Left Wrist</li><li>Right Wrist</li><li>Ankle</li></ul>||Study/site shall be configured in order to utilize this field
 
 **Additional Notes:**
 
-- Depending on the study/site configuration of subject being added, the **gender**, **dob**, and/or **weight** fields may or may not be allowed. If the fields are allowed, then they will be required. If not allowed, then these fields must be excluded from the JSON request.
+- Depending on the study/site configuration of subject being added, the **gender**, **dob**, and/or **weight**, and/or **height** fields may or may not be allowed. If the fields are allowed, then they will be required. If not allowed, then these fields must be excluded from the JSON request.
```
[the remark this answers is not in the document]
Comment (karuna-bhat) [reply sections/subjects.md]: Corrected the additional notes section.

**Pull request merged**